modules/understanding-static-network-policy-connectivity-diff.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_content-type: CONCEPT
+[id="understanding-static-network-policy-connectivity-diff_{context}"]
+= Understanding static network policy connectivity diff
```

Comment:
I would change this title to something like "Analyzing the differences in network policies" so you're focusing on the user behavior/goal rather than on the command.